rubygem-smart_proxy_dynflow/rubygem-smart_proxy_dynflow.spec

```diff
@@ -6,19 +6,21 @@
 
 Summary: Dynflow runtime for Foreman smart proxy
 Name: rubygem-%{gem_name}
-Version: 0.0.7
+Version: 0.1.3
 Release: 1%{?dist}
 Group: Applications/System
 License: GPLv3
 URL: https://github.com/theforeman/smart_proxy_dynflow
 Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
 Requires: ruby(rubygems)
-Requires: foreman-proxy >= 1.9.0
-Requires: rubygem(dynflow) >= 0.8.6
-Requires: rubygem(dynflow) < 0.9.0
-Requires: rubygem(sqlite3)
-Requires: rubygem(sequel)
+Requires: foreman-proxy >= 1.11.0
+
+%if 0%{?fedora}
+Requires: rubygem-smart_proxy_dynflow_core
+%else
+Requires: tfm-rubygem-smart_proxy_dynflow_core
```

Comment:
It might be worth adding `= %{version}` here so you ensure the two packages are in lock-step with each other in case there are internal API changes, and don't get out of sync - especially considering they have different prefixes, it's possible that somebody updates one without the other.

If so, prefer the `rubygem(..)` form as it doesn't include an epoch, i.e.

    Requires: rubygem(smart_proxy_dynflow_core) = %{version}